R/status.R

```diff
@@ -112,7 +169,9 @@ renv_status_impl <- function(project, libpaths, lockpath, sources, cache) {
     renv_status_check_cache(project)
 
   if (synchronized)
-    writef("* The project is already synchronized with the lockfile.")
+    writef("No problems found.")
+  else
+    writef(c("", "See ?status() for advice on resolving the problems."))
```

Comment:
For some reason I don't like the word 'problems' in this context: maybe instead we can use something like "reconciling this state" or something similar? That is, I don't view the lockfile + library being out of sync as a "problem" but just a thing that happens over the lifetime of a project that needs your decision-making.